### DISCUSSION TITLE
TMS coil positioning in SimNIBS 4.1

### DISCUSSION THREAD

**Discussion by umne** [simnibs/simnibs#456] [8. Simulation workflows]
Hello everyone, 
As I'm starting to use the SimNIBS GUI I encountered yet another problem during the simulation of the induced E-field for TMS.
In our protocol, we use a figure of 8 MagVenture Coil to perform cerebellar stimulation using a predefined MNI coordinate translated into individual space as the target.
So when I add the TMS coil position I need to adjust it 'manually' to reproduce visually what we did during the stimulation session (example 1). Sometimes the orientation is good (example 1), but sometimes the coil orientation is completely shifted for the same coordinates in another subject (example 2). Do you know whether there's a way to define reference coordinates for the coil orientation across subjects, that adapt to the individual head mesh? Or should I just try to adapt visually the orientation of the coil?
Is there a way to modify the coil orientation like in the second example? 

Thank you in advance for your precious insights

![image](https://github.com/user-attachments/assets/f58e4e47-2c8e-4058-9020-ae6b9deed9b6)

![image](https://github.com/user-attachments/assets/ffaf6fa1-ad3d-4c59-ba05-6791fed48bb8)

**Comment by TorgeW** [comment]
Could you enable the scalp so we can see where the position is put, it might be that your head mesh is cut of or there is a air cavity close to the ear.
